OHIF/Viewers
# fix(state synchronization): SEG and RT viewports should keep their before hydration voi

fixes #3410 

### Context

As described in the issue above, when having SEG or RT viewport, and the user clicks on the load/hydrate, the old
viewport VOI LUT is not preserved and is removed. The reason for this is that this action is an "update" 
action, and we only storePresentation state on "set" actions (during disable element). 

### Changes & Results

- refactored the storePresentation into a command so that we can call it from both set and update actions.
- In addition the getPresentationIds should not increment the index at the end of the id if the action is an update (
  if in the same viewportIndex and displaySetInstanceUid we are requesting a new presentation id)

### Testing

- https://deploy-preview-3560--ohif-dev.netlify.app/viewer?StudyInstanceUIDs=1.3.6.1.4.1.14519.5.2.1.1706.8374.643249677828306008300337414785
- 1x2 
- d&d SEG on the right
- Invert the left viewport 
- Load the SEG (hydrate)
- see that the left viewport is inverted (was not before the fix)

### Checklist

#### PR



  semantic-release format and guidelines.

#### Code

  etc.)

#### Public Documentation Updates



  additions or removals.

#### Tested Environment

- [] OS: 
- [] Node version: 
- [] Browser:
  


[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch
